pkg/restapi/vc/operation/models.go

```diff
@@ -67,10 +68,9 @@ type IssueCredentialRequest struct {
 
 // IssueCredentialOptions options for issuing credential.
 type IssueCredentialOptions struct {
-	AssertionMethod    string     `json:"assertionMethod,omitempty"`
-	VerificationMethod string     `json:"verificationMethod,omitempty"`
-	ProofPurpose       string     `json:"proofPurpose,omitempty"`
-	Created            *time.Time `json:"created,omitempty"`
+	AssertionMethod string     `json:"assertionMethod,omitempty"`
```

Comment:
We need to support both AssertionMethod and VerificationMethod for now. This duplication will be removed in a future phase.

Reply:
Will add Verification method back

Reply:
Done